bcl-builder.config.js

```diff
@@ -26,8 +26,8 @@ module.exports = {
     },
     {
       from: ["node_modules/@openeuropa/bcl-theme-default/js/**"],
-      to: path.resolve(outputFolder, "assets/js"),
-      options: { up: true },
+      to: path.resolve(outputFolder, "assets"),
```

Comment:
I dont understand why we copy the js into assets instead of assets/js

Reply:
@drishu it's a way to copy the JS folder content _and subdirectories_ into the assets/js. Today I rebased the branch because slim select 2 was included in BCL 1.8.0, released a day after my commit, so now you can pull the latest changes on the branch and check that the JS directory has JS files and slim-select-2 directory with slim select js file.

Reply:
The correct tag for now should be [0.3501.202412051700](https://www.npmjs.com/package/@openeuropa/bcl-bootstrap/v/0.3501.202412051700)

Reply:
@drishu I made a correct prerelease now: 0.3501.202502031900

Please use this version. In 1.8.0 the slim-select-2 should not be available. I made a release 1.8.2 which corrects this.

Reply:
@drishu there is a newer version done by Tiberiu, with fixed style: 0.3501.202502052130